### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by AlenJakob**
Hey all,

First pull request in my life let it fly.

For serious let me know what do you think about the part of my work

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -19,11 +22,15 @@ export default defineComponent({
...
+  font-family: "Poppins", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 18px;
   text-align: center;
-  background: #ecedf4;
+  color: hsl(257, 27%, 26%);
+  margin-top: 60px;
+  max-width: 1440px;
   font-family: "Poppins", sans-serif;
```
Comment (jundymek) [inline src/App.vue:34]: Duplicate (line 26) - to remove.

File: src/App.vue [hunk 2]
```diff
@@ -19,11 +22,15 @@ export default defineComponent({
...
 #app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
+  margin: 0 auto;
+  font-family: "Poppins", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 18px;
   text-align: center;
-  background: #ecedf4;
+  color: hsl(257, 27%, 26%);
```
Comment (jundymek) [inline src/App.vue:31]: There is `_variables.scss` file with colors:
write it: `color: $darkViolet`

File: src/components/SearchBar/Input.vue [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    align-self: flex-start;
+    font-size: 12px;
+    color: #be617a;
+  }
+  & .search {
+    margin-top: 12.5px;
+    display: flex;
+
+    & .input {
+      color: hsl(255, 11%, 22%);
```
Comment (jundymek) [inline src/components/SearchBar/Input.vue:62]: use variables

File: src/components/SearchBar/Input.vue [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+.input {
+  cursor: pointer;
+  border-radius: 8px;
+  margin: 0 8px;
+}
+.txt-warning {
+  margin-left: 5px;
+}
+.btn {
+  background: hsl(180, 66%, 49%);
```
Comment (jundymek) [inline src/components/SearchBar/Input.vue:91]: use variables

File: src/App.vue (hunk 2 above)
Comment (AlenJakob) [reply src/App.vue:31]: I am not sure about that we can use it.
I have once problem that IE doesn't support css variables

Comment (jundymek) [reply src/App.vue:31]: Flexbox also isn't fully supported by IE. It doesn't matter. I think it doesn't matter...

**Pull request merged**